modules/nf-core/custom/dumpsoftwareversions/templates/dumpsoftwareversions.groovy

```diff
@@ -0,0 +1,132 @@
+String dedent(String input) {

```

Comment:
Doesn't groovys native `.stripIndent()` do exactly this?

See also: http://docs.groovy-lang.org/latest/html/documentation/index.html#_triple_single_quoted_string